hpx/lcos/dataflow.hpp

```diff
@@ -477,6 +570,22 @@ namespace hpx { namespace lcos { namespace detail
             return dataflow_dispatch<launch>::call(
                 launch::async, std::forward<F>(f), std::forward<Ts>(ts)...);
         }
+
+        template <
+            typename Allocator, typename F, typename ...Ts,
+            typename Enable = typename std::enable_if<
+                !traits::is_action<typename std::decay<F>::type>::value
+            >::type>
+        HPX_FORCEINLINE static auto
+        call_alloc(Allocator const& alloc, F && f, Ts &&... ts)
+//         ->  decltype(dataflow_dispatch<launch>::call_alloc(
+//                 alloc, launch::async, std::forward<F>(f),
+//                 std::forward<Ts>(ts)...))
```

Comment:
any reason this is commented? It seems that this line is problematic on older clangs.

Reply:
Sorry, it's a left-over. Will fix.